README.md

```diff
@@ -88,27 +68,23 @@ The service name represents the application or component that is producing trace
 
 `--data 'config.service_name=your-preferred-name'`
 
-This value can use vault references. By default, the value of the environment variale `DD_SERVICE` is used by resolving `vault://env/dd-service`.
-When this variable is not set, the default is `kong`.
+The value set through the environment variable `DD_SERVICE` overrides any other specified value, including the default setting.
 
 ### Environment
 
-The environment is a larger grouping of related services, such as `prod`, `staging` or `dev`.
+The environment is a larger grouping of related services, such as `prod`, `staging` or `dev`. By default, generated spans will not have an `environment` tag.
```

Comment:
`environment` is not a tag, though to customers the distinction is probably not important. Consider `s/tag/property/`.

Reply:
I'm wrong, it is a tag. [ref](https://github.com/DataDog/datadog-agent/blob/7550117fa81ba6c32ed04d82e1fa3d99f30b79f7/pkg/proto/datadog/trace/span.proto#L22)